addons/mass_mailing/static/src/views/mass_mailing_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="mailing_contact_views" xml:space="preserve">
+    <t t-name="MailingContactListView.buttons" t-inherit="web.ListView.Buttons" t-inherit-mode="primary" owl="1">
+        <xpath expr="//t[@t-if='props.showButtons']">
```

Comment:
No "position" here?

Button seems there anyway, I tested, but not sure how it works, maybe there is a default.

Reply:
Indeed there's a default value, but I'll provide it explicitly :)